Networking/Networking/Remote/SitePluginsRemote.swift

```diff
@@ -0,0 +1,30 @@
+import Foundation
+import Alamofire
+
+/// SitePlugins: Remote Endpoints
+///
+public class SitePluginsRemote: Remote {
+
+    /// Retrieves all of the `SitePlugin`s for a given site.
+    ///
+    /// - Parameters:
+    ///   - siteID: Site for which we'll fetch the plugins.
+    ///   - completion: Closure to be executed upon completion.
+    ///
+    public func loadPlugins(for siteID: Int64, completion: @escaping ([SitePlugin]?, Error?) -> Void) {
```

Comment:
nit: It could be better to use the return type `Result<[SitePlugin], Error>` here, which seems perfect for this use case and that we started to use by some months in our codebase.

Reply:
Good idea! Fixed in https://github.com/woocommerce/woocommerce-ios/pull/3844/commits/8e320fa88e27122e36d57a3de6e396bc6ec0ce37